x-pack/plugin/monitoring/src/main/resources/monitoring/watches/elasticsearch_cluster_status.json

```diff
@@ -47,19 +47,8 @@
                           }
                         },
                         {
-                          "bool": {
-                            "should": [
-                              {
-                                "term": {
-                                  "_type": "cluster_state"
```

Comment:
This filter was being used for legacy documents to ease migration through 6.x. However, we should not see such documents in 6.7 onwards. So it's safe to remove this filter in 7.0 onwards IMO.

Reply:
To check I understand, all documents will have an explicit `type` field starting in 6.7, so we don't need this extra 'or' clause that matches on document type?

Reply:
Correct. In fact, documents in `.monitoring-*` should've had an explicit type field much before `6.7` but I'm not sure exactly when that was introduced (@pickypg?).

Reply:
Any document created in 5.5+ should have `_type` of `doc` and, `cluster_state` in particular will no longer exist (even as a `type` since `cluster_state` and `cluster_stats` were merged together).

So, of interest to the Watch, the source cluster creates the watch, therefore it would appropriately handles its own data. As a result, this is safe to change, but it means that if someone just upgraded from 5.4 to 5.6 to 6.7, then it would effectively ignore the pre-5.5 data once they got to 6.7 with that type dropped. Given its purpose, that's safe.